app/controllers/lectures_controller.rb

```diff
@@ -343,12 +343,10 @@ def lecture_params
                         :annotations_status, :emergency_link,
                         :emergency_link_status]
       if action_name == "update" && current_user.can_update_personell?(@lecture)
-        allowed_params.concat([:teacher_id, { editor_ids: [] }])
-      end
-      if action_name == "create"
-        allowed_params.concat([:course_id, :teacher_id, { editor_ids: [] }])
+        allowed_params.push(:teacher_id, { editor_ids: [] })
       end
       allowed_params.push(:course_id, :teacher_id, { editor_ids: [] }) if action_name == "create"
+      allowed_params.push(:course_id, :teacher_id, { editor_ids: [] }) if action_name == "create"
```

Comment:
You duplicated the line here

Reply:
Don't know, how this happened (this line is not even related to the Annotation feature). I deleted it.